src/CoreBundle/Response/ResponseVariableReplacerInterface.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/*
+ * This file is part of the Chameleon System (https://www.chameleonsystem.com).
+ *
+ * (c) ESONO AG (https://www.esono.de)
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace ChameleonSystem\CoreBundle\Response;
+
+use ChameleonSystem\CoreBundle\Security\AuthenticityToken\TokenInjectionFailedException;
+
+interface ResponseVariableReplacerInterface
```

Comment:
We usually add a doc comment to the interface declaration describing an overall purpose.

Reply:
done